oak-core/src/test/java/org/apache/jackrabbit/oak/AbstractSecurityTest.java

```diff
@@ -89,8 +89,8 @@ public abstract class AbstractSecurityTest {
     protected Root root;
 
     protected QueryEngineSettings querySettings;
-    private final RootProvider rootProvider = new RootProviderService();
-    private final TreeProvider treeProvider = new TreeProviderService();
+    protected final RootProvider rootProvider = new RootProviderService();
```

Comment:
please use getRootProvider() in the conflict-handler test instead of altering the access modifier